bin/lint-commit-message

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# Find the number of the most recently opened pull request merged so far.
+# Add 1 to this to get the smallest possible number of the next pull request,
+# then add an arbitrary buffer to allow for issues and pull requests opened
+# since the creation of the most recently opened pull request merged so far.
+# This provides an estimate of the number that will appear in the summary of
+# the merge commit so its length can be taken into account.
+
+pull_request_number() {
+  {
+    git log --grep '^Merge pull request #[1-9]' --pretty=format:%s \
+    | cut -d '#' -f 2 \
+    | cut -d ' ' -f 1 \
+    | sort --numeric-sort --reverse ;
+    echo 0 ;
+  } \
+  | head -n 1 \
+  | xargs expr 1 + 25 +
+}
+
+project_name() {
+  git config --get remote.origin.url \
+  | sed -e 's,^git@github[.]com:,,' -e 's,^https://github[.]com/,,' \
```

Comment:
That's a nice way to avoid escape-hell (`\\\\.`) in this case. I will remember. :D